package-lock.json

```diff
@@ -1,8 +1,3406 @@
 {
   "name": "cagov-cron",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Whoa.  Lots of new packages.  What's going on here?

Reply:
npm has an issue we need to discuss as a team (and decide on what upgrades we want to do with our npm packges) & then we can summarize what's going on - it has to do with npm versions & needing to upgrade to the new version of npm

Reply:
I grabbed package-lock from master because I didn't intend to update packages here